extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/destination/DataSourceMSQDestination.java

```diff
@@ -35,7 +36,7 @@
 
 public class DataSourceMSQDestination implements MSQDestination
```

Comment:
Is there a reason to not move MSQDestination to `org.apache.druid.msq.indexing.destination` as well?